urlResolvers/github.js

```diff
@@ -0,0 +1,42 @@
+const URLResolver = require('./urlResolver');
+const GitHubAPIHandler = require('../sourceHandlers/github');
+
+const stripHttp = /^(http(s)?:\/\/)?(www.)?/;
+const urlRegex = /^github\.com\/([\w,-]+)\/([\w,-]+)/;
+
+const apiBaseUrl = 'https://api.github.com/';
```

Comment:
Oh, so you want people to eventually be able to pass in their own GitHub Enterprise API URL to the constructor. That makes sense. But it still seems to me like we should decouple anything having to do specifically with the API from the GitHub resolver and put it directly with the GitHub API handler.

What do you think of handling it as a "default" in the GitHub API handler's constructor? Something along the lines of
``` js
constructor(repoPath, baseUrl) {
    this.repoPath = repoPath;
    if (baseUrl) {
        super(baseUrl);
    }
    else {
        super('https://api.github.com/');
    }
}
```